meson.build

```diff
@@ -1,12 +1,12 @@
 project('libwpe', 'cpp', 'c',
-	meson_version: '>=0.49',
+	meson_version: '>=0.55',
```

Comment:
I know people around are still using Yocto Dunfell as base system, and that comes with Meson 0.53.2—we can probably do this bump given that the next stable release won't be before September this year, hoping that remaining users have migrated by then to the next release, which includes 0.55.1 🤔

Reply:
Since the wrap is only used on Windows, you may not feel the urgency of bumping the version as older versions of Meson will simply be unable to use that wrap.

Again, you can also try to submit the subproject wrap as an upstream PR so that you no longer need to locally maintain an overlay requiring Meson 0.55.0.

Reply:
also, the `can_allow_fallback` wouldn't make sense if the version bump is done.

And not only wraps are used in windows, but also potentially `xkbcommon` in linux.

Reply:
That's a good point. Could you update the patch to also drop the cehcks on `can_allow_fallback` now that we will depend on Meson 0.55? Thanks!